src/views/termsofuse/termsofuse.jsx

```diff
@@ -0,0 +1,514 @@
+var React = require('react');
+var render = require('../../lib/render.jsx');
+var Page = require('../../components/page/www/page.jsx');
+var Box = require('../../components/box/box.jsx');
+
+var Termsofuse = React.createClass({
```

Comment:
@technoboy10 I think that's a good idea. Would this require renaming the branch?